cfme/infrastructure/provider/scvmm.py

```diff
@@ -38,9 +38,14 @@ def _form_mapping(self, create=None, **kwargs):
 
     def deployment_helper(self, deploy_args):
         """ Used in utils.virtual_machines """
+        values = {}
```

Comment:
this code seems to be begging for using a backport of https://docs.python.org/3/library/collections.html#collections.ChainMap

```
combined = ChainMap(deploy_args, self.data, {"host_group": "All Hosts", "cpu": 0, "ram": 0))
return { "hoost_group: combined["host_group"], ...}
```

the example is not tested and may be considered for future use instead of implementing right now